sources/p987.6/sections/00-preface_content.adoc

```diff
@@ -0,0 +1,18 @@
+
+[abstract]
+== Abstract
+
+Key discussion points covered in the draft are stated here in a few
+complete sentences, using passive rather than active voice. The more
+specific the better since the abstract often populates search engines
+and catalog databases.
+
+== Introduction
+
+The introduction of the frontmatter is informative. It serves to give
+readers context, including background, key themes, history, etc.
+
+[.preface]
```

Comment:
"Acknowledgements" are automatically generated, so no need to include here.

Reply:
Apparently, it needs to be marked up; according to the [documentation](https://github.com/metanorma/metanorma.org/pull/611/files#diff-e7160534c607a83e4cdf30c6e42c963f2f947b1938ebd8844aac3f40bd0b4afaR19).
More details in: https://github.com/metanorma/metanorma-ieee/issues/76

Reply:
Actually you're right. I got confused. Thanks.